src/CMakeLists.txt

```diff
@@ -16,7 +16,7 @@
 #
 ###############################################################################
 
-set(LIB_SRCS type.cpp ir_value.cpp enums.h hash.cpp hash.h)
+set(LIB_SRCS type.cpp ir_value.cpp enums.h hash.cpp hash.h data.cpp data.h)
```

Comment:
Let's have files in alphabetical order, one per line.
Like here: https://github.com/ispc/ispc/blob/master/CMakeLists.txt#L218

Reply:
Done